community/runc/APKBUILD

```diff
@@ -22,7 +22,7 @@ builddir="$srcdir/src/github.com/opencontainers/runc"
 options="!check"
 
 # secfixes:
-#   1.0.0_rc8-r2:
+#   1.0.0_rc9:
```

Comment:
not sure it needs update because previous version has this already fixed, but as I understand it needs another rebuild

Reply:
related to the `go-1.13.1` rebuild you mentioned above?

Reply:
I guessed)